tasks/main.yml

```diff
@@ -82,8 +83,14 @@
   vars:
     task_conf_file: "{{ clamd_configuration_path }}"
     task_conf_parameters: "{{ clamav_clamd_configuration }}"
+    task_conf_service: clamd
   when: clamav_clamd_configuration
 
+# We need to flush handlers as configuration changes might be necessary
+# to pass the wait_for signatures test.
+- name: Flush possible pending handlers
+  ansible.builtin.meta: flush_handlers
+
```

Comment:
I'd prefer this be moved to just before the "Wait for new signatures to be downloaded and installed by freshclam" task.

Reply:
This is not accurate. As I have accepted your modification to restart the service no matter what, we need to flush the handlers before the systemd task, which will set the attended status of services, and stop them if we need to.

If we flush after systemd, clamd will always be started if there is a configuration change.

Reply:
@jsf9k what do you think of my previous comment ?